source/php/Component/Acceptance/acceptance.json

```diff
@@ -19,6 +20,18 @@
        "host" : "host ex. youtube.com",
        "name" : "name of the host"
     },
+    "types": {
+        "labels": "array|boolean",
+        "height": "boolean|string",
+        "src": "boolean|array",
+        "policy": "boolean",
+        "host": "boolean",
+        "name": "boolean",
+    	"options": "boolean",
+        "icon": "string",
+        "cover": "boolean",
+        "modifier": "string"
```

Comment:
I see that a lot of the available values (default) are missing a description. Please add the missing ones to description